docs/system/container-metrics.md

```diff
@@ -0,0 +1,108 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Container
+--->
+
+# Semantic Conventions for Container Metrics
+
+**Status**: [Experimental][DocumentStatus]
+
+## Container Metrics
+
+### Metric: `container.cpu.utilization`
+
+This metric is optional.
+
+<!-- semconv metric.container.cpu.time(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `container.cpu.time` | Counter | `ns` | Total CPU time consumed [1] |
+
+**[1]:** Total CPU time consumed by the specific container.
+<!-- endsemconv -->
+
+<!-- semconv metric.container.cpu.time(full) -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `container.cpu.state` | string | The CPU state for this data point. A container SHOULD be characterized _either_ by data points with no `state` labels, _or only_ data points with `state` labels. | `user`; `kernel` | Opt-In |
+| [`container.id`](../attributes-registry/container.md) | string | Container ID. Usually a UUID, as for example used to [identify Docker containers](https://docs.docker.com/engine/reference/run/#container-identification). The UUID might be abbreviated. | `a3bf90e006b2` | Recommended |
+
+`container.cpu.state` MUST be one of the following:
```

Comment:
Those values are defined by docker, but I can imagine this can change going forward, or another container runtime defines other values. So I don't think we should use MUST here